app/models/utils/load_testing/dsl.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Utils
+  module LoadTesting
+    # Extends the JMeter DSL with helper methods
+    class Dsl < RubyJmeter::ExtendedDSL
+      def login(username, password)
```

Comment:
I implemented this before realizing I needed basic auth instead.  It's not currently used but I figured I'd leave it around since we'll likely use it in the future.